File: iterableapi/src/main/java/com/iterable/iterableapi/util/IOUtils.java
package com.iterable.iterableapi.util;

import androidx.annotation.Nullable;

import java.io.Closeable;
import java.io.IOException;

public final class IOUtils {
    private IOUtils() {
    }

    public static void closeQuietly(@Nullable Closeable closeable) {
        if (closeable != null) {
            try {
                closeable.close();
            } catch (IOException ignored) {
            }
        }
    }
}

File: iterableapi/src/main/java/com/iterable/iterableapi/util/IOUtils.kt
package com.iterable.iterableapi.util

import androidx.annotation.Nullable

import java.io.Closeable
import java.io.IOException

object IOUtils {
    fun closeQuietly(closeable: Closeable?) {
        closeable?.let {
            try {
                it.close()
            } catch (ignored: IOException) {
            }
        }
    }
}
